scripts/check_compliance.py

```diff
@@ -252,7 +252,10 @@ def parse_kconfig(self):
         os.environ['GENERATED_DTS_BOARD_CONF'] = "dummy"
 
         # For multi repo support
-        get_modules(os.path.join(tempfile.gettempdir(), "Kconfig.modules"))
+        try:
+            get_modules(os.path.join(tempfile.gettempdir(), "Kconfig.modules"))
+        except sh.CommandNotFound as e:
+            self.skip("Command not found: " + str(e))
```

Comment:
Could add a comment along the lines of `# Skip test if west isn't installed` maybe, to make it clearer what it's about.

Reply:
Agree it wouldn't hurt... on the other hand I suspect no one will look at these lines unless they saw the "Skipped because west's missing" message first. For the rare person who didn't see that message, a quick look at the short get_modules() function shows only one invocation of sh: `sh.west`

I will add that comment if this quick fix requires more time because other, more important problems are found.

Reply:
In the second and newer version `sh.west` is now on the line literally before `sh.CommandNotFound`.